src/Symfony/Component/Security/Csrf/TokenGenerator/UriSafeTokenGenerator.php

```diff
@@ -38,7 +38,16 @@ public function generateToken()
         // Generate an URI safe base64 encoded string that does not contain "+",
         // "/" or "=" which need to be URL encoded and make URLs unnecessarily
         // longer.
-        $bytes = random_bytes($this->entropy / 8);
+        $length = $this->entropy / 8;
+        if (!\is_int($length)) {
+            throw new \InvalidArgumentException(sprintf('Length should be an int, "%s" given.', \gettype($length)));
+        }
+
+        if ($length <= 0) {
+            throw new \InvalidArgumentException('Length should be greater than 0.');
```

Comment:
length is not something the end user knows about. Entropy is. This should be checked in the constructor also.